.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -18,18 +18,13 @@ assignees: ''
 - [ ] Sign off burn-in; convey burn-in results in #besu-release in Discord
 - [ ] Using the same git sha, create a calver tag for the FULL RELEASE, example format `24.4.0`
 - [ ] Using the FULL RELEASE tag, create a release in github to trigger the workflows. Once published:
-    - makes the release "latest" in github
     - this is now public and notifies subscribed users
+    - makes the release "latest" in github
     - publishes artefacts and version-specific docker tags
     - publishes the docker `latest` tag variants
-- [ ] Draft homebrew PR
-- [ ] Draft documentation release
-- [ ] Ensure binary SHAs are correct on the release page
-- [ ] Docker release startup test:
-    - `docker run hyperledger/besu:<version>`
-    - `docker run hyperledger/besu:<version>-arm64`
-    - `docker run --platform linux/amd64 hyperledger/besu:<version>-amd64`
-    - `docker run --pull=always hyperledger/besu:latest` (check version is <version>)
-- [ ] Merge homebrew PR
-- [ ] Publish Docs Release
+- [ ] Check binary SHAs are correct on the release page
+- [ ] Check "Container Verify" GitHub workflow has run successfully
+- [ ] Create homebrew release - run https://github.com/hyperledger/homebrew-besu/actions/workflows/update-version.yml
+- [ ] Create besu-docs release - https://github.com/hyperledger/besu-docs/releases/new
```

Comment:
do you want to add in here that you should copy the changelog from Besu

Reply:
done